plutus-metatheory/src/Raw.lagda

```diff
@@ -92,6 +99,7 @@ decTermCon (bool b) (bool b') with b Data.Bool.≟ b'
 ... | yes p = true
 ... | no ¬p = false
 decTermCon unit unit = true
+-- FIXME: not sure what to do about the BLS types here.
```

Comment:
This function is currently used by NEAT tests, to see if the term returned by two evaluations of the a generated term are the same (agda-plc vs plc, agda-plc vs uplc, or different modes within agda-plc). As long as NEAT doesn't generate results with these new values, it shouldn't matter.

Reply:
Do these types have a comparison function in Haskell?